ThunderCatsTests/UINavigationControllerTests.m

```diff
@@ -0,0 +1,143 @@
+//
+//  UINavigationControllerTests.m
+//  ThunderCats
+//
+//  Created by Kyle Bashour on 7/28/15.
+//  Copyright (c) 2015 Metova. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+//#import <OCMock/OCMock.h>
+#import "UINavigationController+TCAdditions.h"
+
+@interface UINavigationControllerTests : XCTestCase
+
+@property UINavigationController *navigationController;
+@property UIViewController *rootViewController;
+@property UIViewController *middleViewController;
+@property UIViewController *topViewController;
+@property NSArray *pushedViewControllers;
+
+@end
+
+@implementation UINavigationControllerTests
+
+- (void)setUp {
+    [super setUp];
+    
+    self.rootViewController = [UIViewController new];
+    self.middleViewController = [UIViewController new];
+    self.topViewController = [UIViewController new];
+    self.navigationController = [[UINavigationController alloc] initWithRootViewController:self.rootViewController];
+    
+    UIWindow *navWindow = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+    [navWindow setRootViewController:self.navigationController];
+    [UIApplication sharedApplication].keyWindow.rootViewController = self.navigationController;
+    
+    [self.navigationController popToRootViewControllerAnimated:NO];
+    [self.navigationController pushViewController:self.middleViewController animated:NO];
+    [self.navigationController pushViewController:self.topViewController animated:NO];
+    self.pushedViewControllers = [[NSArray alloc] initWithObjects:self.middleViewController, self.topViewController, nil];
+}
+
+
+- (void)tearDown {
+    
+    [super tearDown];
+}
+
+//- (void)testPushViewController {
```

Comment:
Do we even need this file?  Everything is commented out... nothing is tested here.